<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>512</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>631</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>THULAC</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>设置</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>请选择模型位置：</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>91</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>选择</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>160</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>您选择了：</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>351</width>
          <height>61</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="pathLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>210</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>请设置分词分隔符：</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_7">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>210</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QPlainTextEdit" name="paraText">
           <property name="plainText">
            <string>_</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <zorder>verticalLayoutWidget_3</zorder>
       <zorder>pushButton</zorder>
       <zorder>verticalLayoutWidget_4</zorder>
       <zorder>verticalLayoutWidget_5</zorder>
       <zorder>verticalLayoutWidget_6</zorder>
       <zorder>verticalLayoutWidget_7</zorder>
       <zorder>paraText</zorder>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>分词</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit">
            <property name="plainText">
             <string>请输入文本</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="isFilter">
            <property name="text">
             <string>使用过滤器</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="isT2S">
            <property name="text">
             <string>转换为简体</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="isSegOnly">
            <property name="text">
             <string>词性标注</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>478</width>
               <height>102</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="buttonRun">
          <property name="text">
           <string>分词</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>关于</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="icons.qrc">:/1.png</pixmap>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="styleSheet">
             <string notr="true">color: rgb(38, 68, 255)</string>
            </property>
            <property name="text">
             <string>THULAC：一个高效的中文词法分析工具包.
THULAC: An Efficient Lexical Analyzer for Chinese.
2016</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>作者：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Maosong Sun （孙茂松，导师）</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Xinxiong Chen（陈新雄，博士生）</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Kaixu Zhang (张开旭，硕士生）</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Zhipeng Guo（郭志芃，本科生）</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Xingyu Pan（潘星宇，本科生）</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Zhiyuan Liu（刘知远，助理教授）</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="styleSheet">
             <string notr="true">color : red
</string>
            </property>
            <property name="text">
             <string>Copyright：Natural Language Processing and 
Computational Social Science Lab</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_15">
            <property name="styleSheet">
             <string notr="true">color : red
</string>
            </property>
            <property name="text">
             <string>版权所有：清华大学自然语言处理与社会人文计算实验室</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>552</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDivi">
   <property name="text">
    <string>分隔符</string>
   </property>
   <property name="toolTip">
    <string>分隔符</string>
   </property>
  </action>
  <action name="actionModule">
   <property name="text">
    <string>模型路径</string>
   </property>
   <property name="toolTip">
    <string>模型路径</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
